docs/src/pages/WebRatingPage/WebRatingPage.js

```diff
@@ -187,6 +187,41 @@ const components = [
       </div>,
     ],
   },
+  {
+    id: 'pill',
+    title: 'Pill style ratings',
+    blurb: [
+      <Paragraph>The rating component&apos;s have a `pill` style.</Paragraph>,
```

Comment:
```suggestion
      <Paragraph>The rating components have a `pill` style.</Paragraph>,
```

Reply:
I can change the components but the putting quotes around pill `eslint` does not like so will just remove any quotes

Reply:
Ah fair enough